pkg/cfn/manager/cluster.go

```diff
@@ -18,7 +18,12 @@ func (c *StackCollection) CreateCluster(errs chan error, _ interface{}) error {
 		return err
 	}
 
-	return c.CreateStack(name, stack, c.spec.Metadata.Tags, nil, errs)
+	// All the tags contained in `c.spec.Metadata.Tags` are already contained in the shared tags.
+	// Providing it to CreateStack results in duplicated stack tags.
+	// See https://github.com/weaveworks/eksctl/issues/413 for more details.
+	tags := map[string]string{}
+
+	return c.CreateStack(name, stack, tags, nil, errs)
```

Comment:
I think passing `nil` and simple one line comment will do
```suggestion
    // Unlike with `CreateNodeGroup`, all tags are already set for the cluster stack
	return c.CreateStack(name, stack, nil, nil, errs)
```

Reply:
Much simpler, and makes sense much more! Thanks for the suggestion.